### ISSUE TITLE
Reinstall Issues on PopOS

### ISSUE THREAD

**Issue by Grant592** [regolith-linux/regolith-desktop#100]
I had regolith desktop working on PopOs with no major issues however I modified to config file which stopped Rofi from working properly (error message saying couldn't find rasi file). I uninstalled and purged regolith desktop and all other files from regolith repo. I then attempted to delete any other files related to regolith and i3 from both /etc and /home to ensure there would be no clashes when reinstalling. Upon reinstalling, I can login to regolith from splash screen however there is then no functionality just a background and nothing else. Is there a way to delete all files and config files and start completely fresh?

**Comment by kgilmer** [comment]
Hi @Grant592 , I have not seen Regolith get into a state where it only shows wallpaper, and not at least some kind of error.  It maybe be something that Regolith depends on is not working.  If that's the case, then the following steps are not likely to work, as it sounds like you've already done them.  However, here is what I'd do to try and get Regolith back to factory presets:
```
$ # log out of regolith session, log into ubuntu session
$ sudo apt remove regolith-desktop
$ mkdir ~/regolith-backup
$ mv ~/.config/i3-regolith ~/regolith-backup
$ mv ~/.config/regolith ~/regolith-backup
$ mv ~/.Xresources* ~/regolith-backup
$ sudo apt install regolith-desktop
$ # reboot and try logging into the Regolith session
```

If that doesn't work, perhaps look into `/var/log/syslog` and see if during the time that you log into Regolith that you have errors or other clues that may lead to a solution.

Let me know how it goes, we'll get you fixed up!

**Comment by Grant592** [comment]
Hi, thanks for the reply. When i re-installed regolith prior to your reply, it did not seem to create the config files in ~/.config like it did when I installed it the first time therefore the instructions above don't really work. I've tried deleting any residual files as I said previously and re-installing but, again, the install doesn't seem to create the appropriate config files in either /etc or /home

Could you advise what I would be looking for in /var/log/syslog?

**Comment by kgilmer** [comment]
I need more details when you say something doesn't work.  For example, if there is an error, I need to know what the error is.  If a command seems to perform no change, I need that detail as well.  

Regarding `/var/log/syslog` I would search of scan for errors or warnings during the time frame you're experiencing problems.  Hard to know exactly what to look for.

**Comment by Grant592** [comment]
Apologies for the lack of info. I go from the beginning just to try and clarify things.
1. I installed regolith-desktop originally and it worked fine. I made some changes to the config files which seemed to stop rofi from working properly
2. I uninstalled regolith-desktop and deleted any config files and all files associated with i3 and regolith on /etc
3. I reinstalled regolith-desktop and when I logged into the desktop I got a background with no fucntionality
4. I tried to follow your instructions above but the reason they didn't work is that the folders that they refer to did not exist. It's as if when I resinstalled regolith-desktop, it didn't recreate the config files
5. I tried one more uninstall/purge/reinstall and then when I attempted to log in, I got the same situation where I had a background and nothing else except this time I received a pop up saying that gnome-flashback had stopped responding. I can press escape to get rid of the pop up and then the only thing that changes is there is a thick black cross for a cursor but the desktop does not respond to any clicks or keyboard shortcuts meaning I need to long press the power button to restart

Thanks for your time and help on this issue
